package.json

```diff
@@ -70,19 +70,19 @@
     "@graphql-codegen/cli": "^4.0.1",
     "@graphql-codegen/client-preset": "^4.0.1",
     "@graphql-typed-document-node/core": "^3.2.0",
-    "@storybook/addon-actions": "7.4.0",
-    "@storybook/addon-essentials": "7.4.0",
-    "@storybook/addon-interactions": "7.4.0",
-    "@storybook/addon-mdx-gfm": "7.4.0",
-    "@storybook/channels": "7.4.0",
-    "@storybook/client-logger": "7.4.0",
+    "@storybook/addon-actions": "0.0.0-pr-24007-sha-c31c498a",
```

Comment:
Do we have a sense when this API will be available? I'm not sure if we want to be stuck on a canary for a while if we are giving this to customers and the SB release process is back to 6 weeks.

Can we work around (i.e. do a bit more on our end) while the API is not yet available?

Reply:
It's in an alpha now, upgrading...